addons/sale/models/sale_order_line.py

```diff
@@ -667,7 +667,9 @@ def product_id_change(self):
                 }
             }
 
-    @api.depends('product_uom', 'product_uom_qty')
+    @api.depends(
+        'product_uom', 'product_uom_qty', 'product_id',
+        'order_id.partner_id', 'order_id.pricelist_id')
```

Comment:
Nope, we don't want to update the prices of all lines when the pricelist is updated (Cf whole `update_prices` method and flow)

Reply:
Right, thanks :)